tests/test_classifier/test_rocauc.py

```diff
@@ -88,6 +88,60 @@ def test_binary_rocauc(self):
         visualizer.poof()
         self.assert_images_similar(visualizer)
 
+    def test_binary_probability_decision(self):
+        """
+        Test ROCAUC with a binary classifier with both decision & predict_proba
+        """
+        # Create and fit the visualizer
+        visualizer = ROCAUC(AdaBoostClassifier())
+        visualizer.fit(self.binary.X.train, self.binary.y.train)
+
+        # Score the visualizer
+        s = visualizer.score(self.binary.X.test, self.binary.y.test)
+
+        # Test that score method successfully returns a value between 0 and 1
+        assert 0 <= s <= 1
+
+        # Check the scores
+        self.assertEqual(len(visualizer.fpr.keys()), 4)
+        self.assertEqual(len(visualizer.tpr.keys()), 4)
+        self.assertEqual(len(visualizer.roc_auc.keys()), 4)
+
+        for k in (0, 1, "micro", "macro"):
+            self.assertIn(k, visualizer.fpr)
+            self.assertIn(k, visualizer.tpr)
+            self.assertIn(k, visualizer.roc_auc)
+            self.assertEqual(len(visualizer.fpr[k]), len(visualizer.tpr[k]))
+            self.assertGreater(visualizer.roc_auc[k], 0.0)
+            self.assertLess(visualizer.roc_auc[k], 1.0)
+
+        # Compare the images
+        visualizer.poof()
+        self.assert_images_similar(visualizer)
+
+    def test_binary_decision(self):
+        """
+        Test ROCAUC with a binary classifier with a decision_function
+        """
+        # Create and fit the visualizer
+        visualizer = ROCAUC(LinearSVC(random_state=42), micro=False, macro=False, per_class=False)
+        visualizer.fit(self.binary.X.train, self.binary.y.train)
+
+        # Score the visualizer
+        s = visualizer.score(self.binary.X.test, self.binary.y.test)
+
+        # Test that score method successfully returns a value between 0 and 1
+        assert 0 <= s <= 1
+
+        # Check the scores
+        self.assertEqual(len(visualizer.fpr.keys()), 1)
+        self.assertEqual(len(visualizer.tpr.keys()), 1)
+        self.assertEqual(len(visualizer.roc_auc.keys()), 1)
+
+        # Compare the images
+        visualizer.poof()
```

Comment:
The test is failing on CI with an RMSE of 6.8 - I'm wondering if it's a font issue; removing the call to `poof()` simplifies the test and may get it to pass?